canonical-web-and-design/ubuntu.com
# Update contact form url on /security/fips

## Done

- Updated connected forms so the two different forms are used for the `Contact us` button and ` Join our free beta programme for a FIPS-validated OpenJDK `link each. 

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Please refer to [form doc](https://docs.google.com/document/d/1dA6y-0VCsug50cUnWsNsFQYp1-leg8JZ_EU1rCKJeiA/edit#) and [brief](https://docs.google.com/document/d/1dzUObyMPCrlLRNVj7lfjxIG02os1tTUDAxN3IB0UNLo/edit) if needed
1. Go to `/security/fips` [Demo](https://ubuntu-com-10765.demos.haus/security/fips)
2. Click `Contact us` and check if it a modal form shows
3. Click `Join our free beta programme for a FIPS-validated OpenJDK` and check if it goes to `/security/fips/contact-us`
